tex/chProgramming.tex

```diff
@@ -825,14 +825,14 @@ \subsection{Natural numbers}\index[concept]{natural number}
 the inductive type.  For example, the following definition is
 rejected by \Coq{}.
 
-\begin{coq}{name=f_plus_one}{width=6cm,title=Non exhaustive case analysis}
+\begin{coq}{name=f_plus_one}{width=6cm,title=Non-exhaustive case analysis}
 Definition wrong (n : nat) :=
   match n with 0 => true end.
 $~$
 \end{coq}
 \coqrun{name=f_plus_one_run}{f_plus_one}
 \begin{coqout}{f_plus_one_run}{width=6cm,title=Response}
-Error: Non exhaustive pattern
+Error: Non-exhaustive pattern
```

Comment:
I agree with the rule of thumb you implemented and documented. But this one is in an error message output by Coq... I think it is better to reproduce it verbatim, error included. And in parallel, may be try to improve Coq's English :).